campaign_themes/default.json

```diff
@@ -43,7 +43,8 @@
 					],
 					"default": "greenpeace"
```

Comment:
If the assumption is "there is no default theme", then we should remove this file.

Reply:
Well currently this file contains the allowed values for certain style settings if no theme is selected. So it makes sense to have a config file for the default theme (but without "hidden_fields"), else if we wanted to provide these options the default theme would have to store it somewhere in the code in another way than the actual themes.